mercury-framework/examples/README.md

```diff
@@ -30,9 +30,26 @@ The customer credit rating service domain always returns the same value for any
 1. Create a Minikube cluster with the Mercury Operator and all the dependencies. 
 Look at the [Mercury Operator installation instructions](../../mercury-operator/README.md)
 2. Create a Service Domain Cluster
+
+   `minikube start -p service-domain`
+
 3. Create the different Service Domains (Customer Offer, Party Routing Profile and Customer
 Credit Rating)
+
+   `kubectl apply -f deploy/01_deployment.yaml`
+
 4. Initiate a Customer Offer
+   
+    ``curl -v -H "Content-Type: application/json" `minikube service customer-offer-camelk-rest --url=true`/customer-offer/${SD_REFERENCE}/customer-offer-procedure/initiation -d '{"procedure": {"customerReference": "${CUSTOMER_REFERENCE}"}}``
```

Comment:
The integrations and kubernetes services will be created by the operator, check the name of the service